docs/roman/resample/arguments.rst

```diff
@@ -77,10 +77,20 @@ image.
     The weighting type for each input image.
     If `weight_type=ivm` (the default), the scaling value
     will be determined per-pixel using the inverse of the read noise
-    (VAR_RNOISE) array stored in each input image. If the VAR_RNOISE array does
-    not exist, the variance is set to 1 for all pixels (equal weighting).
+    (``VAR_RNOISE``) array stored in each input image. If the ``VAR_RNOISE`` array does
+    not exist, the weight is set to 1 for all pixels (equal weighting).
     If `weight_type=exptime`, the scaling value will be set equal to the
     exposure time found in the image header.
+    If `weight_type=ivsky`, the scaling value will be determined per-pixel
+    using the inverse of the sky variance (``VAR_SKY``) array calculated in the
+    resample step for each input image. ``VAR_SKY`` is given by the following equation:
+
+    .. math::
+
+        \text{VAR\_SKY} = \text{VAR\_RNOISE} + \text{VAR\_POISSON} \, \frac{ med(\text{DATA}) }{ \text{DATA} },
+
+    where :math:`\text{DATA}` and :math:`med(\text{DATA})` correspond to the data array and its median, respectively.
+    If the ``VAR_SKY`` array does not exist, the weight is set to 1 for all pixels (equal weighting).
```

Comment:
Should this say if the `VAR_RNOISE` and/or `VAR_POISSON` array does not exist?

Reply:
Also I'm not sure the escapes are needed for the underscores in the variable names. They may be causing the rendering to look odd in the docs:
https://roman-pipeline--1556.org.readthedocs.build/en/1556/roman/resample/arguments.html#step-arguments

Reply:
Weird, RTD is showing something different. Do you see something like the following:
<img width="605" alt="Screenshot 2025-01-29 at 5 35 32 PM" src="https://github.com/user-attachments/assets/9541fc73-6e46-4c95-843e-e159b1e9b31a" />
at 
https://roman-pipeline--1556.org.readthedocs.build/en/1556/roman/resample/arguments.html#step-arguments

Reply:
Yeah, I noticed that too.

Reply:
Maybe it has something to do with the build command?
- locally, `make html` runs `sphinx-build -b html -d _build/doctrees   . _build/html`
- during CI we run `sphinx -T -W --keep-going -b html -d _build/doctrees -D language=en . $READTHEDOCS_OUTPUT/html`